.github/workflows/build-and-run-model.yaml

```diff
@@ -13,6 +13,36 @@ on:
   pull_request:
     types: [opened, reopened, synchronize, closed]
   workflow_dispatch:
+    inputs:
+      run_note:
+        type: string
+        description: Note to include with run
+        required: true
+      report_additional_pins:
+        type: string
+        description: Create reports for additional PINs
+        default: '13253180150000 05174150240000'
+        required: false
+      upload_enable:
+        type: boolean
+        description: Upload results to S3
+        default: true
+        required: true
+      cv_enable:
+        type: boolean
+        description: Run cross-validation
+        default: false
+        required: true
+      comp_enable:
+        type: boolean
+        description: Run comparables finding
+        default: false
+        required: true
```

Comment:
TBD on whether or not condo comps will actually be a thing. Fine leaving this toggle in for now though.